client/login/wp-login/index.jsx

```diff
@@ -450,7 +448,6 @@ export class Login extends Component {
 			return (
 				<>
```

Comment:
Since it no longer returns multiple components, it seems like the use of Fragment is no longer necessary. Same applies for the other return statements below that have been affected.

Reply:
You're correct.